hack/make-rules/test.sh

```diff
@@ -241,19 +241,19 @@ produceJUnitXMLReport() {
     return
   fi
 
-  local test_stdout_filenames
   local junit_xml_filename
-  test_stdout_filenames=$(ls "${junit_filename_prefix}"*.stdout)
   junit_xml_filename="${junit_filename_prefix}.xml"
-  if ! command -v go-junit-report >/dev/null 2>&1; then
-    kube::log::error "go-junit-report not found; please install with " \
-      "go get -u github.com/jstemmer/go-junit-report"
+
+  if ! command -v gotestsum >/dev/null 2>&1; then
+    kube::log::error "gotestsum not found; please install with " \
+      "go get -u gotest.tools/gotestsum"
```

Comment:
should we be telling people to install from vendor instead?

Reply:
this was mostly a copy/paste of the existing message... given our scripts do the install from vendor, I don't really care either way

Reply:
i figured as much, imho we should suggest doing what we would do, otherwise they're going to run it with a different version? /shrug

Reply:
let's revisit later actually ... @cblecker penny for your thoughts ...